notebooks/00_quick_start/als_pyspark_movielens.ipynb

```diff
@@ -222,7 +223,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Took 3.917855978012085 seconds for training.\n"
+      "Took 1.6378734111785889 seconds for training.\n"
```

Comment:
This is dependent on the VM we use for running the notebook. Do we want to add information to make it more clear?

Reply:
Sure I can add something above this